resync with upstream

TallTed opened the pull request.

megoth committed:
An attempt at making the presentation of paid services clearer

TallTed committed:
clarify which fork to make PRs against

angelo-v committed:
reviewed apps and restructured list

angelo-v committed:
merged gh-pages branch from upstream

angelo-v committed:
reviewed apps and restructured list

angelo-v committed:
merge information from solid w3c wiki

angelo-v committed:
merge historical apps from solid w3c wiki

angelo-v committed:
fixed URL of solid file manager

jeff-zucker committed:
changes to JZ app listings

jeff-zucker committed:
Update README.md

jeff-zucker committed:
Update README.md

angelo-v committed:
Merge pull request #1 from jeff-zucker/master

cleanup some of JZ's listings

Commit:
Merge pull request #34 from angelo-v/master

Cleanup app list and merge info from w3.org solid wiki

Commit:
Update README.md

Commit:
Merge pull request #35 from solid/Mitzi-Laszlo-patch-1

Update README.md

Commit:
Added `LinkedPipes` Applications project

Commit:
Update README.md

Commit:
Merge pull request #39 from solid/Mitzi-Laszlo-patch-2

Update README.md

regisrob committed:
Fix link syntax

michielbdejong committed:
Add Solid Notify app

Commit:
Update README.md

Commit:
Merge pull request #42 from solid/Mitzi-Laszlo-patch-3

Update README.md

davidshq committed:
Update README.md

- Minor formatting edits.
- Fixed Mark Book link, added separate github link.

openlink reviewed (changes requested):
Why the conflicts? I don't have issues with the content, just the merge conflicts.

TallTed commented:
I cannot explain the conflicts.  They're complex enough that github can't display them in the browser interface, and just tells me to use the command line (which is not my usual tool, so it will take some extra time).  <strike>Will try to resolve them.</strike>

I do not understand enough of **`GitHub Desktop.app`** nor the `git` commandline to do this.  Both appear to assume that they've been in active use previous to these conflicts, and the instructions github provides are insufficient to let me see what the conflicts are, never mind to resolve them.

Commit:
Merge pull request #43 from davidshq/gh-pages

Update README.md

Commit:
Merge pull request #41 from solid/michielbdejong-patch-1

Add Solid Notify app

Commit:
Merge branch 'gh-pages' into patch-1

Commit:
Merge pull request #40 from regisrob/patch-1

Fix link syntax

Commit:
Merge branch 'gh-pages' into introducing-emoji-for-paid-services

Commit:
Merge pull request #28 from solid/introducing-emoji-for-paid-services

An attempt at making the presentation of paid services clearer

Commit:
Merge pull request #31 from TallTed/patch-2

clarify which fork to make PRs against

Commit:
Merge pull request #36 from aorumbayev/gh-pages

Added `LinkedPipes Applications` project

Commit:
Update README.md

Commit:
Merge pull request #44 from solid/Mitzi-Laszlo-patch-4

Update README.md

Commit:
Update README.md

Commit:
Merge pull request #45 from solid/Mitzi-Laszlo-patch-5

Update README.md

Commit:
Update README.md

Commit:
Merge pull request #46 from solid/Mitzi-Laszlo-patch-6

Update README.md

Commit:
Update README.md

Included licensing info and licensor as well as year. Listed chronologically with a complete explanation per app rather than per category.

Commit:
Merge pull request #47 from solid/Mitzi-Laszlo-patch-7

Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

TallTed commented:
this PR is moot
